tests/system_tests/test_core/test_wandb_run.py

```diff
@@ -139,7 +130,7 @@ def test_deprecated_feature_telemetry(wandb_backend_spy):
         )
 
 
-def test_except_hook(wandb_init, test_settings):
+def test_except_hook(user, test_settings):
```

Comment:
I didn't get to it as part of the settings refactor, but I think we should remove test_settings for good and instead add an auto-use fixture that would set up two env vars for the default settings that we're flipping.

Reply:
yeah i think we could just completely remove it honestly, without the 2 settings.